docs/features/communication/some_ip_gateway/index.rst

```diff
@@ -0,0 +1,251 @@
+
+The SOME/IP Gateway service instance shall be defined in the deployment configuration.
+
+- :need:`ACL Placement <feat_req__com__acl_placement>`
+- :need:`ACL per Service Instance <feat_req__com__acl_per_service_instance>`
+
+
+Backwards Compatibility
+=======================
+
+As there is currently no previous solution for gateways in S-CORE, no backwards compatibility is required.
+Subsequent changes to the SOME/IP gateway module shall keep the API stable where possible and introduce breaking APIs only with approval from tech lead circle.
+Applications shall stay stable on API layer, need to recompile is acceptable.
+
+Security Impact
+===============
+
+Be aware, communication with SOME/IP generally is considered to be not secure. Integrators may apply measures that are outside the
+scope of the SOME/IP gateway to secure the communication (IPsec, MACsec, ...).
+
+.. note::
+   it is expected that a feature request for crypto and security will cover the necessary measures.
+   `Feature Request for Security & Crypto <https://github.com/eclipse-score/score/issues/905>`_
+
+
+Access Control List (ACL)
+-------------------------
+
+The gateway shall only forward selective service instances originating from IP addresses configured in an allow-list.
+The logic required for this is protocol specific and therefore part of this feature request.
+
+Be aware that an additional security mechanism may be necessary that ensures that IP addresses are not forged. This mechanism is out of scope of this feature.
+
+An access control mechanism is part of a firewall solution, which states that only the traffic defined in the security policy is allowed onto the network and all other traffic must be silently dropped.
+Access Control acts on OSI Layer 5-7. It shall fulfill the following:
+
+- What SOME/IP service instances to forward
+- What IP address is allowed to offer a SOME/IP service instance
+- Configuration of ACL is done at deployment
+- Versioning of services
+- ACL shall be able to be switched on/off
+- It shall be possible to persistently log ACL drop actions
+
+.. note::
+  - Checking SOME/IP-SD messages with the ACL is optional because no functional data is transported.
+  - SOME/IP-SD messages are not protected as per AUTOSAR Adaptive specification.
+
+
+.. uml::
+   :name: doc__acl_activity_diagram
+   :scale: 80
+   :align: center
+   :caption: Simplified Access Control Activity Diagram
+
+   !include assets/puml-theme-score.puml
+
+   start
+
+   :Ingress SOME/IP;
+   :ACL Check;
+
+   if (Packet matches ACL entry?) then (Yes)
+      :Process Packet;
+   else (No)
+      :Log Event;
+      if (ACL is active?) then (Yes)
+         :Drop Packet;
+      else (No)
+         :Process Packet;
+      endif
+   endif
+
+   stop
+
+
+End-to-End (E2E) protection with CRC and counters
+-------------------------------------------------
+
+Applications communicating over the network may have to secure data with end-to-end protection (E2E), which may involve
```

Comment:
The word "secure" gives the impression of this being a security mechanism. But this is only about safety and not about security at all.

```suggestion
Applications communicating over the network may have to protect data with end-to-end protection (E2E), which may involve
```